web/src/pages/dashboard.tsx

```diff
@@ -104,6 +104,7 @@ const DashboardPage = (props: Props): ReactElement => {
     bodyText: Config.dashboardDefaults.opportunitiesAndDeadlinesBody,
     variant: "success",
     dataTestId: "deadlines-opportunities-alert",
+    delayInMilliseconds: 6000,
```

Comment:
[sand] I like the custom delay, but I wonder if this is a sensible value to put into config so that Content can change it later if need be.